### PR TITLE
Add AUR Package

### PR DESCRIPTION
**Pull request by 1995parham**
I have added the documentation about AUR package and also I have added the badge about available packages and their version.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,8 @@ and support full JetStream management.
 
 ### Installation
 
+[![Packaging status](https://repology.org/badge/vertical-allrepos/natscli.svg)](https://repology.org/project/natscli/versions)
```
Comment (ripienaar) [inline README.md:21]: since this really only apply to Arch, I think it's best to remove this line, else this is good thanks :)

**Comment by 1995parham** [comment]
Yes, you are right. I have removed it.

**Comment by ripienaar** [comment]
thank you

**Comment by ripienaar** [comment]
May I ask how this updates on the Arch side? It's automatic or you have to maintain it?

**Comment by 1995parham** [comment]
Arch repositories are like the brew one but there isn't any automatic way for them. For now you can mention me in each release but go releaser has plan for automating it.

**Pull request merged**